controllers/request.controller.js

```diff
@@ -49,7 +59,11 @@ exports.create = async (req, res, next) => {
 exports.getScheduleResquests = async (req, res, next) => {
   try {
     const schedule = req.schedule._id;
-    let requests = await Request.getBy({ schedule });
+    let requests = [];
+    // check if schedule is closed
+    if (req.schedule.isClosed)
+      return res.json(sendResponse(httpStatus.OK, 'The schedul is closed', requests));
```

Comment:
You have a typo here
```suggestion
      return res.json(sendResponse(httpStatus.OK, 'The schedule is closed', requests));
```